### DISCUSSION TITLE
Adding aqara opple mx960

### DISCUSSION THREAD

**Discussion by Winfast2003** [Koenkk/zigbee2mqtt#6934] [Devices]
Add a cheer to the official support for aqara opple mx960 lamp. I cannot connect it. There is mx560, mx450 and my lamp is not there.

**Comment by Elkropac** [comment]
Hi, did you manage to pair it? If so, how? I just enabled "Permit all" and plugged in power on MX960, i cannot see it in interface

**Reply by Elkropac** [reply]
Ah ok, with google translate on chinese manual, i found out, i have to turn it on and off for 5 times
